config.yaml

```diff
@@ -41,7 +41,7 @@ build_iso:
     mock_args:
     packages_dir: "result/packages/latest"
     automated_install_file: "host-os.ks"
-    host_os_packages_groups:
+    installable_environments:
```

Comment:
why you removed "host_os" prefix? I'd prefer to differ the things which belong to Open Power Host OS and the ones which belong to Cent OS (base distro) eg the groups below